library/arangodb

```diff
@@ -1,12 +1,15 @@
 # maintainer: Frank Celler <info@arangodb.com> (@fceller)
 # maintainer: Wilfried Goesgens <w.goesgens@arangodb.org> (@dothebart)
+# maintainer: Vadim Kondratyev <vadim@arangodb.org> (@KVS85)
```

Comment:
Just an FYI to @fceller and @dothebart.

Reply:
Thanks! The guy is described here: https://www.arangodb.com/about-arangodb/.

Reply:
hey @yosifkit - yes, Vadim is overseeing the ArangoDB release management. Sorry we didn't introduce him properly ;-)